packages/run-protocol/src/reserve/assetReserve.js

```diff
@@ -90,7 +90,7 @@ const start = async (zcf, privateArgs) => {
   };
 
   const makeAddCollateralInvitation = () =>
-    zcf.makeInvitation(addCollateralHook, 'Add Collateal');
+    zcf.makeInvitation(handleAddCollateralOffer, 'Add Collateal');
```

Comment:
I would have expected either
   * `'Collect Fees`' and `'Add Collateral'`, or
   * `'CollectFees'`, and `'AddCollateral'`